src/styles/overlay-panel.less

```diff
@@ -16,6 +16,7 @@
   border: 1px solid @color-pf-black-400;
   height: 50%;
   left: 20px;
+  min-height: 350px;
```

Comment:
Can't say I disagree, just that I had heard it from designers that they wanted to see two sizes available for the panel. Let's leave this for now and continue discussions as the designs firm up.